docs/user/openstack/README.md

```diff
@@ -242,43 +242,50 @@ All the OpenShift nodes get created in an OpenStack tenant network and as such,
 
 This method allows you to attach two floating IP (FIP) addresses to endpoints in OpenShift.
 
-You will need to create two floating IP addresses, one to attach to the API load balancer (lb FIP), and one for the OpenShift applications (apps FIP). Note that the LB FIP is the same floating IP as the one you added to your `install-config.yaml`. The following command is an example of how to create floating IPs:
+A standard deployment uses three floating IP addresses in total:
 
-```sh
-openstack floating ip create <external network>
-```
+1. External access to the OpenShift API
+2. External access to the workloads (apps) running on the OpenShift cluster
+3. Temporary IP address for bootstrap log collection
 
-You will also need to add the following records to your DNS:
+The first two addresses (API and Ingress) need to be created up-front and have the corresponding DNS records resolve to them.
```

Comment:
do they? I thought only the API one needs to exists before hand. The ingress one can be created after the cluster is installed, right?

Reply:
That is true. However, it really complicates the docs for no practical benefit :-(.

I'll try to update the wording so it's less absolute without complicating things.